scripts/e2e-cursor-live.sh

```
@@ -0,0 +1,175 @@
+fi
+
+if ! "${ARCHON[@]}" version >/dev/null 2>&1; then
+  fail "archon CLI not available. Run: cd packages/cli && bun link"
+fi
+pass "archon CLI: $("${ARCHON[@]}" version 2>/dev/null | head -1)"
+
+# Load user-scoped secrets (CLI path)
+if [[ -f "$ARCHON_HOME/.env" ]]; then
+  set -a
+  # shellcheck disable=SC1090
+  source "$ARCHON_HOME/.env" 2>/dev/null || true
```

Shellcheck (0.11.0)

[info] Not following: ./.env was not specified as input (see shellcheck -x).

(SC1091)